CONTRIBUTING.md

```diff
@@ -53,6 +53,7 @@ Only admins need to worry about these step.
 1. Make sure you have a clean working tree (`git status` should show no changes)
 1. `npm ci`
 1. `npm run lint:check`
+1. Update `CHANGELOG.md` with a summary of your changes
 1. `npm version`
```

Comment:
Change to `npm version n.n.n` or something to indicate a version # should be supplied.

Reply:
Updated!